composer.json

```diff
@@ -17,8 +17,9 @@
   },
   "require-dev": {
     "phpunit/phpunit": "~6.0|^7.0|^8.0|^9.0",
-    "orchestra/testbench": "^3.5|^4.0|^5.0|^6.0",
-    "orchestra/database": "^3.5|^4.0|^5.0|^6.0"
+    "orchestra/testbench": "^6.17",
+    "orchestra/database": "^3.5|^4.0|^5.0|^6.0",
+    "nunomaduro/collision": "^5.3"
```

Comment:
The reason why there are several versions of `orchestra/testbench` listed as requirements is because we still need older versions of it to test integration with older versions of Laravel (done in a Github Action). Luckily, it's API didn't change so it's possible to do it.